### PR TITLE
Duality adapter

### PR DESCRIPTION
**Pull request by NicholasDotSol**

### CODE REVIEW

**Comment by thal0x** [comment]
There is a merge conflict, looks like it's just related to us changing the name from Skip API -> Skip Go. Once this is resolved I can approve 🙏

File: packages/skip/src/swap.rs [hunk 1]
```diff
@@ -47,6 +47,12 @@ pub struct LidoSatelliteInstantiateMsg {
     pub lido_satellite_contract_address: String,
 }
 
+#[cw_serde]
+pub struct DualityInstatiateMsg {
```
Comment (NotJeremyLiu) [inline packages/skip/src/swap.rs:51]: Instantiate is misspelled

**Comment by NicholasDotSol** [comment]
> There is a merge conflict, looks like it's just related to us changing the name from Skip API -> Skip Go. Once this is resolved I can approve 🙏

fixed

**Comment by NotJeremyLiu** [comment]
@NicholasDotSol Can you resolve the failing CI for clippy and format

**Pull request merged**